### PR TITLE
README update for GA

### PR DESCRIPTION
**Pull request by KunJeongPark**
### Summary of changes

- Remove language of “in development/not yet available” 

### Checklist

~- [ ] Added a changelog entry~

### Authors
- @KunJeongPark

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Ven
...
+This product is fully open source. We welcome any and all feedback.
 
 ### Contribution
-As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
+We will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you 
+would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. 
+If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
```
Comment (scannillo) [inline README.md]: I would just leave this part:

```suggestion
If you have suggestions for features that you would like to see in the SDK, please feel free to open an issue, PR, or discussion with suggestions.
```

File: README.md [hunk 2]
```diff
@@ -4,10 +4,12 @@ Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Ven
 
 ## FAQ
 ### Availability
-The SDK is currently in the development process. This product is being developed fully open source - throughout the development process, we welcome any and all feedback. Aspects of the SDK _will likely_ change as we develop the SDK. We recommend using the SDK in the sandbox environment until an official release is available. This README will be updated with an official release date once it is generally available.
+This product is fully open source. We welcome any and all feedback.
```
Comment (scannillo) [inline README.md]: I think this sentence can move into the `Contribution` section, and when we can delete the `Availability` section

**Comment by scannillo** [comment]
Also whatever changes come as a result of this PR should be equally applied to Android. We want to make the language the same.

**Review by sshropshire — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -75,7 +73,7 @@ GitHub Actions CI will run all tests and build commands per package manager on e
 
 ### Local Testing
 
-TBD until development progresses. We will use Rake, Fastlane, or some other tool for easy command-line build tasks.
+We use Rake, Fastlane, or some other tool for easy command-line build tasks.
```
Comment (jaxdesmarais) [inline README.md]: Looks like we just use fastlane, we don't have a rakefile:
```suggestion
We use [Fastlane](https://docs.fastlane.tools/getting-started/ios/setup/) for command-line build tasks.
```

File: README.md [hunk 4]
```diff
@@ -3,11 +3,9 @@
...
 
 ## FAQ
-### Availability
-The SDK is currently in the development process. This product is being developed fully open source - throughout the development process, we welcome any and all feedback. Aspects of the SDK _will likely_ change as we develop the SDK. We recommend using the SDK in the sandbox environment until an official release is available. This README will be updated with an official release date once it is generally available.
-
 ### Contribution
-As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
+If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. 
+This product is fully open source. We welcome any and all feedback.
```
Comment (scannillo) [inline README.md:8]:
```suggestion
This product is fully open source. We welcome any and all feedback.
```

Comment (scannillo) [inline README.md]:
```suggestion
If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. This product is fully open source. We welcome any and all feedback.
```

Comment (KunJeongPark) [reply README.md]: What is the difference here Sammy? Is it all on same line?

Comment (scannillo) [reply README.md]: Yup, just moving it to the same line.

**Review by scannillo — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -75,7 +72,7 @@ GitHub Actions CI will run all tests and build commands per package manager on e
 
 ### Local Testing
 
-TBD until development progresses. We will use Rake, Fastlane, or some other tool for easy command-line build tasks.
+We use [Fastlane](https://docs.fastlane.tools/getting-started/ios/setup/) for command-line build tasks.
```
Comment (scannillo) [inline README.md:75]: We really should have instructions for how to use fastlane to run our tests locally. This can be it's own ticket.

**Pull request merged**